gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiPlansResource_ListTest.java

```diff
@@ -43,16 +39,10 @@
 import io.gravitee.rest.api.service.v4.PlanSearchService;
 import jakarta.ws.rs.core.Response;
 import java.util.*;
-import org.junit.After;
-import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mockito;
 import org.springframework.beans.factory.annotation.Autowired;
 
-public class ApiPlansResource_GetPlansTest extends AbstractResourceTest {
-
-    private static final String API = "my-api";
-    private static final String ENVIRONMENT = "my-env";
+public class ApiPlansResource_ListTest extends ApiPlansResourceTest {
```

Comment:
Shouldn't we use the name of the java method? Here it would be:
```suggestion
public class ApiPlansResource_GetApiPlans extends ApiPlansResourceTest {
```

Reply:
Not against to be honest but definitely, GetApiPlans looks not very aligned with other test class names 😕
Also, when developing the tests, it was much more difficult for me to quickly distinguish it from the simple get plan test.
I can rename it another way but could you provide a clean naming convention to follow? For example, other test classes don't appear very well named:

- ApiResource_getApiByIdTest
- ApiResource_StartTest
- ApiResource_exportApiDefinitionTest

Reply:
As discussed together, we will start writing our dev conventions, and then after we will be able to make a refactor PR to apply the naming convention.

So for the moment, you can keep it like that.